bonus.md

```diff
@@ -0,0 +1,64 @@
+
+![arena](img/arena-ocean-basin.jpg)
+<p class="image-caption"> Top view of the ocean basin arena. </p>
+
+
+Since the basin is a research facility, and the area is not open to the public, only a small number of the team members will be allowed to enter the basin. The rest of the SAUVC participants will have to watch the attempt from the viewing gallery. Details on the number of team members allowed to enter the basin will be released at a later date.
+
+#### Starting Zone
+
+The starting zone is a **140×140cm area** marked on the surface of water. The teams **must** start their AUVs from this area. The team may only place the AUV at the water surface, it must autonomously submerge **before** leaving the starting zone.
+
+### Surfacing
+
+Breaching the surface any point during the mission causes <span class="emphasis">end of the current attempt</span>. The team may wish to retry if they still have time left.
+
+#### Points
+
+There will be no points awarded for the bonus round. The teams will be ranked based on the time taken to the <span class="emphasis">end of the attempt</span> ONLY IF the team successfully managed to complete the task. The team with the fastest time will be ranked first for the bonus round.
+
+#### Specification of props
+
+The gate is located in the middle of the basin, ~25m from the starting zone. The gate is 200cm tall and 200cm wide. The gate is marked with <span class="indicate-red">red</span> and <span class="indicate-green">green</span> stripes on the port and starboard sides respectively.
+
+
+|   |       Prop       |              Description                |
+|---|:----------------:|:---------------------------------------:|
+| 1 | Gate             | 200cm wide and 200cm tall gate with <span class="indicate-red">red</span> and <span class="indicate-green">green</span> markings on port and starboard sides respectively. |
+| 2 | Orange Flare     | 150cm tall and ~15cm diameter flare, and <span class="indicate-orange">orange</span> in color. |
+
+
+![flare](img/bonus-gate.jpg)
+<p class="image-caption"> Front-view of gate</p>
+
+![flare](img/orange-flare.jpg)
+<p class="image-caption"> Front-view of the orange flare. </p>
+
+#### Gameplay (Bonus round)
+
+Only the top teams from the final round will be allowed to attempt the bonus round. Ranking for the bonus round will be tabulated separately from the main competition. The top teams in the bonus round will receive a special bonus prize regardless of their performance in the main competition.
+
+There will be NO practice time for the bonus round. The teams will be given a time slot to attempt the bonus round. The teams will be given 15 minutes to attempt the bonus round. The teams may attempt the bonus round as many times as they wish within the 15 minutes time slot. The team may choose to abort the attempt at any time. However, the retreival of the AUV might take some time because of the logistics involved.
```

Comment:
The sentence "The teams will be given a time slot to attempt the bonus round." can be deleted, as the next sentence covers it.

Also, typo: retreival -> retrieval